packages/reports/app/styles/navi-reports/components/report-builder.less

```diff
@@ -23,6 +23,14 @@
     padding: 5px 10px;
   }
 
+  &__container--disabled {
+    background: @denali-gray-300;
```

Comment:
Can we set `cursor: not-allowed;`;

Reply:
is there a `cursor: banished-to-void-realm;`?

Reply:
And `pointer-events: none;`

Reply:
What do you guys think about showing the selected metrics and dimensions when the report is run? Then when the report finishes running, the metric and dimension selectors will go back to whatever state they were in before running.